runtime/planner.js

```diff
@@ -142,6 +149,10 @@ export class Planner {
       });
     }
   }
+  dispose() {
+    this._relevances.forEach(relevance => relevance.newArc.dispose());
```

Comment:
Add a comment on why we need to explicitly dispose (can probably paraphrase Shane's comment from the conversation).

Reply:
done